controllers/app.controllers.js

```diff
@@ -1,7 +1,13 @@
 const { fetchTopics } = require("../models/app.models");
+const endpoints = require("../endpoints.json");
 
 exports.getTopics = (req, res, next) => {
   fetchTopics()
     .then((topics) => res.status(200).send({ topics }))
     .catch(next);
 };
+
+exports.getApi = (req, res, next) => {
+  console.log(endpoints);
```

Comment:
Don't forget to clean up the console logs once you've finished with an endpoint 🧹